core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -55,7 +55,7 @@ var PreferencesPage = function() {
   var customProfilePhoto = element(
     by.css('.protractor-test-custom-photo'));
   var profilePhotoCropper = element(
-    by.css('.protractor-test-photo-crop'));
+    by.css('.cropper-container'));
```

Comment:
Can we add a `protractor-test-*` class to a parent (or grandparent, etc.) element of the croppable iamge and then look for `.cropper-container` within that element's children? For example:

```js
element(by.css('.protractor-test-photo-crop').element(by.css('.cropper-container'))
```

The `protractor-test-*` name is a pretty strong convention in the e2e tests, so it might be confusing for devs if we don't include it at all. For example, someone might assume that because no `protractor-test-*` classes exist on an element or its parents, they can change it without messing up any e2e tests. Does this make sense?